### PR TITLE
Added check to wait untill label value is reflected in API.

### PR DESCRIPTION
**Pull request by amitbhatt818**
- Added check for label in control and data plane install test case

Signed-off-by: Amit Bhatt <amitbhatt818@gmail.com>

### CODE REVIEW

File: litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml [hunk 1]
```diff
@@ -226,47 +226,53 @@
...
-            url_username: "{{ username.stdout}}"
-            url_password: "{{ password.stdout}}"
-            force_basic_auth: yes
-            return_content: yes
-            body_format: json
-            status_code: 200
-          register: node3
-          failed_when: "node3.json.nodeType['mayadata.io/control-plane'] != 'true'"
+        ## Checking the data plane label on the first node                                                                                                                                                          
+        - name: Checking the label on the first node of the cluster
```
Comment (harshshekhar15) [inline litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml:230]: Rename the task to - `Checking data plane label on the first node of the cluster`

File: litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml [hunk 2]
```diff
@@ -226,47 +226,53 @@
...
+            status_code: 200                                                                                                                                                                                            
+          register: node1                                                                                                                                                                                               
+          until: "'mayadata.io/data-plane' in {{node1.json.labels | to_json | string }}"                                                                                                                                
+          delay: 2                                                                                                                                                                                                      
+          retries: 60
+
+        ## Checking the control plane label on the second node                                                                                                                                                          
+        - name: Checking the label on the second node of the cluster
```
Comment (harshshekhar15) [inline litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml:246]: Rename the task to - `Checking control plane label on the second node of the cluster`

File: litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml [hunk 3]
```diff
@@ -226,47 +226,53 @@
...
+            status_code: 200                                                                                                                                                                                            
+          register: node2                                                                                                                                                                                               
+          until: "'mayadata.io/control-plane' in {{node2.json.labels | to_json | string }}"                                                                                                                                
+          delay: 2                                                                                                                                                                                                      
+          retries: 60             
+
+        ## Checking the control plane label on the third node                                                                                                                                                           
+        - name: Checking the label on the third node of the cluster
```
Comment (harshshekhar15) [inline litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml:262]: Rename the task to - `Checking control plane label on the third node of the cluster`

File: litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml [hunk 4]
```diff
@@ -249,47 +249,53 @@
...
+            force_basic_auth: yes                                                                                                                                                                                       
+            return_content: yes                                                                                                                                                                                         
+            body_format: json                                                                                                                                                                                           
+            status_code: 200                                                                                                                                                                                            
+          register: node1                                                                                                                                                                                               
+          until: "'mayadata.io/control-plane' in {{node1.json.labels | to_json | string }}"
```
Comment (harshshekhar15) [inline litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml:264]: @amitbhatt818 I guess this condition will now work fine without converting the node1 json. Can you please verify this once?

Comment (amitbhatt818) [reply litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml]: Yes @harshshekhar15 It is working fine without json also.

**Pull request merged**